setup.py

```diff
@@ -109,14 +110,15 @@ def get_custom_rc_file(self):
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: APLv2 License',
+        'License :: OSI Approved :: Apache Software License',
```

Comment:
Dumb question from the ignorant: What is APLv2 and Apache Software License? Neither of them appear listed here: https://www.apache.org/licenses/

@mattsfuller: Can you confirm what this should be? You spend more time talking to the lawyers than we do.